views/about.hbs

```diff
@@ -13,14 +13,19 @@
   </div>
 </div>
 <div class="container-fluid text-center">
-  <div class="team-row text-center">
-    <div class="col-sm-10">
+  <div class="team-row">
+    <div class="col-sm-10" id="team">
       {{#each team}}
-      <div class="team-box">
+      <div class="team-box" data-toggle="collapse" data-target="#{{id}}" onclick="(e) => {console.log(e);}">
```

Comment:
@jarjan can you please review this? My Bootstrap knowledge is very basic :)

Reply:
All good, except onclick is unnecessary. 
https://getbootstrap.com/docs/3.4/javascript/#collapse

Reply:
Thanks, I deleted it. I used onclick for debugging and forgot to remove it afterwards.